torchao/dtypes/nf4tensor.py

```diff
@@ -393,9 +401,13 @@ def convert_to_norm_float_weight(
         scaled_blocks = blocks / scales
 
         # Returns a flattened tensor with each element quantized to nf4 index
-        quantized_blocks = NF4Tensor.quantize_tensor_nearest(
-            scaled_blocks.flatten(), nf4
-        )
+
+        quantized_blocks = torch.empty(numel, dtype=torch.uint8)
```

Comment:
nit: add a comment for the reasoning behind this

Reply:
I added a note above but let me actually reference here